expression/distsql_builtin_test.go

```diff
@@ -920,8 +920,8 @@ func newIntFieldType() *types.FieldType {
 
 func newDurFieldType() *types.FieldType {
 	return &types.FieldType{
-		Tp:   mysql.TypeDuration,
-		Flag: types.DefaultFsp,
+		Tp:      mysql.TypeDuration,
+		Decimal: int(types.DefaultFsp),
```

Comment:
a bugfix?

Reply:
I think so.
But nothing is affected by this line. 😢